src/manipulation-planner.cc

```diff
@@ -242,14 +243,21 @@ namespace hpp {
       SuccessStatistics& es = edgeStat (edge);
       if (!edge->applyConstraints (n_near, qProj_)) {
         HPP_STOP_TIMECOUNTER (applyConstraints);
+        es.addFailure (reasons_[FAILURE]);
         es.addFailure (reasons_[PROJECTION]);
         return false;
       }
+      if (qProj_.isApprox (*q_near)) {
```

Comment:
This test should be using the distance function since opposed quaternion part of configurations must be compared equals.